src/components/BooksList.js

```diff
@@ -16,6 +16,10 @@ const BookList = () => {
   // const listBooks = useSelector((store) => {
   //   return store.books.array;
   // });
```

Comment:
- [ ] [OPTIONAL] Please kindly remove commented code as already pointed out by the other reviewer. I only included this as optional because it is the only feedback that is unattended but please make sure it is resolved